manifests/cf-manifest/spec/manifest/release_versions_spec.rb

```diff
@@ -98,7 +93,7 @@ def normalise_version(v)
     expect(monitor_remote_smoke_tests_resource_version).to eq(cf_smoke_tests_resource_version)
   end
 
-  specify "cf-acceptance-tests version should be the same as the CF manifest version" do
+  skip "cf-acceptance-tests version should be the same as the CF manifest version" do
```

Comment:
Instead of skipping the test please add an if inside it, so we accept 7.6 as a valid branch for cf-acceptance-tests if the version is 7.7. If we remove this test we might forget about bumping the version the next time.

Reply:
I've added a commit that addresses this with an if statement. It's now only skipped if the cf-deployment version is v7.7.0, in which case it expects the branch to be cf7.6.

Should fail go back to the old behaviour on the next upgrade.